src/mongo_client.ts

```diff
@@ -641,17 +654,44 @@ export class MongoClient extends TypedEventEmitter<MongoClientEvents> implements
   }
 
   /**
-   * Cleans up client-side resources used by the MongoClient.
+   * Cleans up resources managed by the MongoClient.
+   *
+   * The close method clears and closes all resources that the client is responsible for.
+   * Please refer to the `MongoClient` class documentation for a breakdown of resource responsibilities.
```

Comment:
Is this documented somewhere?  I don't see it in the doc comment for MongoClient.

Reply:
I think Neal is referring to L360, the list under "It handles a multitude of features on your application's behalf: ..."

Reply:
updated based on our conversation